e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs

```diff
@@ -20,7 +20,7 @@ namespace IIoTPlatform_E2E_Tests.Orchestrated
     /// </summary>
     [TestCaseOrderer("IIoTPlatform_E2E_Tests.TestExtensions.TestOrderer", TestConstants.TestAssemblyName)]
     [Collection("IIoT Platform Test Collection")]
-    [Trait("IsApiTest", "true")]
+    [Trait("PublisherMode", "orchestrated")]
```

Comment:
can you add both name and values to the TestConstants?

Reply:
Could enum work better here ?

Reply:
Done.